### ISSUE TITLE
Jitter test b68872 fails due to exception thrown in exception filter.

### ISSUE THREAD

**Issue by janvorli** [dotnet/coreclr#1969]
This test throws an exception from an exception filter and the exception handler code asserts because m_uCatchToCallPC is 0, and we pass that as the handler PC to CallHandler.
Here are some details on the test:

This runs the code at TRY1 with filter at FILTER_1 and handler at HANDLER_1:
```IL
.try TRY_1 to FILTER_1 filter FILTER_1 handler HANDLER_1 to END_1
 ```

The TRY_1 throws System.Exception unconditionally:
```IL
newobj instance void [mscorlib]System.Exception::.ctor()
throw
```
That means that in the first pass of the exception handling, it invokes the FILTER_1. The filter gets the exception as its parameter (will be on top of the stack of the IL virtual machine):
```IL
FILTER_1:
            isinst [mscorlib]System.Single
            ldstr      "inside 1st filter"
            call       void [System.Console]System.Console::WriteLine(string)
          
            unbox float32
 ```
The isinst above produces null, since the exception is not of type System.Single. And the last line above – the unbox – throws a null reference exception when trying to unbox float32 from NULL.

Exceptions in exception filters should get swallowed and the exception filter that throws should just be ignored, as if it returned "false".

**Comment by adityamandaleeka** [comment]
As part of the effort to get CI test runs up, #1928 includes adding b68872 to the disabled test list. This is just a reminder to remove that test from the list when this is fixed.

**Comment by janvorli** [comment]
Fixed by #2031
